streaming/text/enwiki.py

```diff
@@ -68,7 +71,18 @@ def __getitem__(self, idx: int) -> Any:
             Any: Sample data.
         """
         obj = super().__getitem__(idx)
+        if 'segment_ids' in obj.keys():
+            obj['token_type_ids'] = obj.pop('segment_ids')
+        if 'input_mask' in obj.keys():
+            obj['attention_mask'] = obj.pop('input_mask')
+        obj.pop('next_sentence_labels')
+        obj.pop('masked_lm_weights')
         for key, value in obj.items():
             dtype = self.field_dtypes[key]
             obj[key] = np.frombuffer(value, dtype)
+        input_len = len(obj['input_ids'])
+        obj['labels'] = np.full((input_len,), -100)
+        obj['labels'][obj['masked_lm_positions']] =  obj['masked_lm_ids']
+        obj.pop('masked_lm_positions')
+        obj.pop('masked_lm_ids')
```

Comment:
Would it be simpler to return a new dict over changing in-place?

Reply:
What do you prefer? I can do either.

Reply:
Created a new dict.